core/aws_ddk_core/stages/__init__.py

```diff
@@ -12,12 +12,14 @@
 # See the License for the specific language governing permissions and
```

Comment:
Could you also please add an `AppFlowStage` entry [here](https://github.com/awslabs/aws-ddk/blob/main/docs/source/core/aws_ddk_core.rst) in the `DataStages` section

Reply:
added